### ISSUE TITLE
Any plan to release the configurations that can reproduce the ImageNet/COCO results?

### ISSUE THREAD

**Issue by lcmeng** [BayesWatch/pytorch-blockswap#1]
It'll be great to try out something other than CIFAR10. TIA.

**Comment by jack-willturner** [comment]
I've added them now (under `genotypes/bs_imagenet_N.csv`), as well as a ResNet model definition to `models/resnet.py`.

**Comment by jack-willturner** [comment]
I'll need to do some testing and then I'll close this issue:

- [ ] double check the genotypes are correct
- [ ] add an imagenet/coco training script
- [ ] include a seed for reproducing the results
- [ ] add the baselines (downscaled resnets)

**Comment by lcmeng** [comment]
Thank you. I'll give it a spin as soon as you think it's in good shape. BTW, the README uses`--param_goal` which is no longer a valid argument for model_generator.py.
